### ISSUE TITLE
Verify that adapter sequences are correct for podar dataset

### ISSUE THREAD

**Issue by brooksph** [dahak-metagenomics/dahak#10]
Truseq?

**Comment by brooksph** [comment]
We should modify the [.settings](https://github.com/dahak-metagenomics/dahak-taco/blob/master/rules/read_filtering/read_filtering.settings) file downloading adapter sequences step to allow for the use of user specified adapter sequences. This can be done by create a config yaml or json (e.g. https://github.com/common-workflow-language/workflows/blob/master/tools/trimmomatic-illumina_clipping.yaml) file in the same manner we will use to allow the user to specify sequences.

**Comment by charlesreid1** [comment]
Yes, the file you linked to defines a default parameters dictionary for read filtering. That means each of the parameters in that dictionary can be overridden by the user, so they can add this to their workflow parameters .json to use a custom adapter at an arbitrary URL:

```
        'adapter_file' : {
            'MyAdapter.fa' : 'http://example.com/some/fancy/new/adapter/file.fa'
        }
```
